website/source/docs/guides/leader-election.html.md

```diff
@@ -1,130 +1,141 @@
 ---
 layout: "docs"
-page_title: "Leader Election"
+page_title: "Application Leader Election with Sessions"
 sidebar_current: "docs-guides-leader"
 description: |-
   This guide describes how to build client-side leader election using Consul. If you are interested in the leader election used internally to Consul, please refer to the consensus protocol documentation instead.
 ---
 
-# Leader Election
+# Application Leader Election with Sessions
 
-This guide describes how to build client-side leader election using Consul. If you
-are interested in the leader election used internally by Consul, please refer to the
-[consensus protocol](/docs/internals/consensus.html) documentation instead.
+For some applications, like HDFS, it is necessary to set one instance as
+a leader. This ensures the application data is current and stable.
 
-There are a number of ways that leader election can be built, so our goal is not to
-cover all the possible methods. Instead, we will focus on using Consul's support for
-[sessions](/docs/internals/sessions.html). Sessions allow us to build a system that
-can gracefully handle failures.
+This guide describes how to build client-side leader elections for service 
+instances, using Consul. Consul's support for
+[sessions](/docs/internals/sessions.html) allows you to build a system that can gracefully handle failures.
 
--> **Note:** JSON output in this guide has been pretty-printed for easier reading. Actual values returned from the API will not be formatted.
+If you
+are interested in the leader election used internally by Consul, please refer to the
+[consensus protocol](/docs/internals/consensus.html) documentation instead.
 
-## Contending Nodes
+## Contending Service Instances 
 
-Let's imagine we have a set of nodes who are attempting to acquire leadership
-for a given service. All nodes that are participating should agree on a given
+Imagine you have a set of service instances who are attempting to acquire leadership
+for a given service. All service instances that are participating should agree on a given
 key to coordinate. A good pattern is simply:
 
 ```text
 service/<service name>/leader
 ```
 
-We'll abbreviate this pattern as simply `<key>` for the rest of this guide.
+Let's abbreviate this pattern as simply as a `<key>` for the rest of this guide.
+
+```text
+service/dbservice/leader = lead
```

Comment:
I got a bit lost here when following along. I think this assignment could benefit from being called out a bit more explicitly in words. As in, clarifying that the guide will use `lead` as `<key>`.

Also, a slightly less generic key like `db-lead` might be a clearer abbreviation for when that key pops up later in the guide.

Reply:
I actually think we could remove this not all together. I think this is more confusing than just using the full key `service/dbservice/leader`, especially since the full key really isn't too long.

Reply:
True, that sounds good to me.

Reply:
I think we've resolved this confusion